File: suwan/Programmers/step1/x만큼 간격이 있는 n개의 숫자.py
def solution(x, n):
    return [x + (i * x) for i in range(n)]


x1 = 2
n1 = 5  # return [2,4,6,8,10]

x2 = 4
n2 = 3  # return [4,8,12]

x3 = -4
n3 = 2  # return [-4,-8]

print(solution(x1, n1))
print(solution(x2, n2))
print(solution(x3, n3))

File: mmkser10/javascript/programmers/x만큼_간격이_있는_n개의_숫자.js
function solution(x, n) {
    var answer = [];
    for(let i = 0 ; i < n ; i++){
        answer[i] = x*(i+1);
    }
    return answer;
}

// another sol
function anotherSolutionA(x, n) {
    return Array(n).fill(x).map(((value, index) => (index+1)*value));
}